astropy/io/ascii/basic.py

```diff
@@ -16,29 +16,32 @@
 from . import core
 
 class BasicHeader(core.BaseHeader):
-    '''Basic table Header Reader
+    """
```

Comment:
1. I don't think single or double quotes matter here as long as they are consistent in that particular docstring, but I do prefer double quotes myself.
2. I don't think new line right after the quotes matter as well, i.e., it is a personal preference. But I am not against it.